From 450a4a4584d65777c6d6638f9f99175cdc85297d Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Tue, 31 Jan 2023 10:05:27 +0200
Subject: [PATCH] [SW-115443] gaudi2: credo fw upgrade

return the new credo fw to 1.8.1

This reverts commit 3603cc683d36be40fb802c93d66e41f818f33058.
---

diff --git a/board/habanalabs/gaudi2/fw/condor.fw.bin b/board/habanalabs/gaudi2/fw/condor.fw.bin
index e8977ad..f8f6613 100644
--- a/board/habanalabs/gaudi2/fw/condor.fw.bin
+++ b/board/habanalabs/gaudi2/fw/condor.fw.bin
Binary files differ
